interface/patient_file/summary/add_edit_amendments.php

```diff
@@ -122,18 +123,12 @@
 
 <html>
 <head>
-<?php html_header_show();?>
-
-<!-- supporting javascript code -->
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-1.7.2.min.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
-
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery.datetimepicker.full.min.js"></script>
-
+<?php
+    html_header_show();
+    call_required_libraries(array('jquery-min-3-1-1', 'datepicker'));
```

Comment:
Please add comment "requires textformat.js and dialog.js" or something to indicate that we need those unspecified libraries.

Reply:
I can do it, but is it needed for the whole refactor? I mean, this function ALWAYS requires those 2 scripts... is this the only place that needs that comment?